Solar_System_create.sql

```diff
@@ -0,0 +1,145 @@
+-- Created by Vertabelo (http://vertabelo.com)
+-- Last modification date: 2022-09-11 19:06:34.215
+
+-- tables
+-- Table: battery
+CREATE TABLE battery (
+    type varchar(10) NOT NULL,
+    capacity int NOT NULL,
+    duration_time int NOT NULL,
+    charge_level varchar(5) NOT NULL,
+    power int NOT NULL,
+    CONSTRAINT battery_pk PRIMARY KEY (type)
+);
+
+-- Table: business_land
+CREATE TABLE business_land (
+    id int NOT NULL AUTO_INCREMENT,
+    quantity_station int NOT NULL,
+    address varchar(20) NOT NULL COMMENT 'e.g. street, region',
+    city_name varchar(15) NOT NULL,
+    CONSTRAINT business_land_pk PRIMARY KEY (id)
+);
+
+-- Table: city
+CREATE TABLE city (
+    name varchar(15) NOT NULL,
+    country_name varchar(15) NOT NULL,
+    CONSTRAINT city_pk PRIMARY KEY (name)
+);
+
+-- Table: country
+CREATE TABLE country (
+    name varchar(15) NOT NULL,
+    CONSTRAINT country_pk PRIMARY KEY (name)
+);
+
+-- Table: element
+CREATE TABLE element (
+    quantity int NOT NULL COMMENT 'e.g battery, panel',
+    panel_type varchar(10) NOT NULL,
+    battery_type varchar(10) NOT NULL,
+    panel_quantity int NULL,
+    battery_quantity int NULL,
+    CONSTRAINT element_pk PRIMARY KEY (quantity)
+) COMMENT '
+';
+
+-- Table: energy
+CREATE TABLE energy (
+    solar_amount int NOT NULL,
+    use_now int NOT NULL,
+    exporting int NULL,
+    energy_market_id int NOT NULL,
+    CONSTRAINT energy_pk PRIMARY KEY (solar_amount)
+);
+
+-- Table: energy_market
+CREATE TABLE energy_market (
+    id int NOT NULL AUTO_INCREMENT,
+    price double(10,2) NOT NULL,
+    date date NOT NULL,
+    time time NOT NULL,
+    CONSTRAINT energy_market_pk PRIMARY KEY (id)
+);
+
+-- Table: owner
+CREATE TABLE owner (
+    id int NOT NULL AUTO_INCREMENT,
+    name varchar(20) NOT NULL,
+    surname varchar(30) NOT NULL,
+    email varchar(20) NOT NULL,
+    password varchar(20) NOT NULL,
```

Comment:
Ok, typically you don't store the password in your DB, you store a salt (salt is an encrypted version of password that takes much longer to crack, and when a user logs in, you encrypt the password they provided and compare it with the salt stored) or use an external auth system (e.g. Firebase Auth, AWS Cognito etc.). That's not a big deal for this task, just so you know: you shouldn't do it in your real database :)